infra/bootstrap/account/main.tf

```diff
@@ -0,0 +1,57 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+locals {
+  # Profile that will be used to select which account to deploy to.
+  profile = "staging"
+  # Choose the region where this infrastructure should be deployed.
+  region = "us-east-1"
+  # Set project tags that will be used to tag all resources. 
+  tags = {
+    project     = "template-application-nextjs"
+    environment = "staging"
+    owner       = "platform"
+    repository  = "https://github.com/navapbc/template-application-nextjs"
+    description = "Application resources"
+  }
```

Comment:
Is it worth pulling these into a separate `constants` or `common` module? We do this for PFML and then add onto them like this, which makes them easy to change in one spot:

```py
tags = merge(module.common.tags, {
  environment = "staging"
})
```

Reply:
I think so!